pkg/clients/aws/cloudformation/cloudformation.go

```diff
@@ -89,10 +89,18 @@ func IsErrorNotFound(err error) bool {
 
 // IsCompletedState validates that operation is complete for a create or update.
 func IsCompletedState(status cf.StackStatus) bool {
-	return status == cf.StackStatusCreateComplete || status == cf.StackStatusUpdateComplete
+	switch status {
+	case cf.StackStatusCreateComplete, cf.StackStatusUpdateComplete:
+		return true
+	}
+	return false
 }
 
 // IsFailedState reports if the operation has failed and can not continue.
 func IsFailedState(status cf.StackStatus) bool {
-	return status == cf.StackStatusCreateFailed || status == cf.StackStatusRollbackComplete || status == cf.StackStatusRollbackFailed || status == cf.StackStatusDeleteComplete || status == cf.StackStatusDeleteFailed
+	switch status {
+	case cf.StackStatusCreateFailed, cf.StackStatusRollbackComplete, cf.StackStatusRollbackFailed, cf.StackStatusDeleteComplete, cf.StackStatusDeleteFailed:
+		return true
+	}
+	return false
```

Comment:
`IsFailedState` and `IsComplededState` do put off a utility function smell.  The functions have no logic, so I can buy-in on  the map.  Would you prefer the `var` in the `_sync` function which is the only place these functions are called, or at the package level?

This also gets us simplifies (in the tiniest) the `client/` packages.

Reply:
I don't feel super strongly either way, but I'd lean a tiny bit toward making it a private package level variable given it's static.

Reply:
Pushed in d9b6666.

You can see that this goes against some original convention because `git grep github.com/aws` is rare (breaking convention?) outside of the pkg/clients directory.